applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -117,6 +118,12 @@ def AddVariables(self):
     def AddDofs(self):
         KratosMultiphysics.VariableUtils().AddDof(KCPFApp.VELOCITY_POTENTIAL, self.main_model_part)
         KratosMultiphysics.VariableUtils().AddDof(KCPFApp.AUXILIARY_VELOCITY_POTENTIAL, self.main_model_part)
+        KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.PRESSURE, self.main_model_part)
```

Comment:
Hi @marcnunezc the reason why @BasselSaridar added this variables is because the embedded test is currently failing when ran in FullDebug becuase of https://github.com/KratosMultiphysics/Kratos/issues/4834. I just talked with @jcotela and he told me that this will be fixed as soon as https://github.com/KratosMultiphysics/Kratos/pull/5066 is merged.

@BasselSaridar as soon as https://github.com/KratosMultiphysics/Kratos/pull/5066 is merged you can merge master and remove the PRESSURE and VELOCITY_X...

Reply:
@BasselSaridar now you are able to remove pressure and velocities from the dof list.